lib/imageResize.ts

```diff
@@ -21,17 +35,30 @@ export const ImageResize = Image.extend({
         view,
         options: { editable },
       } = editor;
-      const { style } = node.attrs;
+      const { wrapperStyle, containerStyle } = node.attrs;
+      const inline = this.options.inline;
+      const display = inline ? 'inline-block' : 'block';
       const $wrapper = document.createElement('div');
       const $container = document.createElement('div');
       const $img = document.createElement('img');
       const iconStyle = 'width: 24px; height: 24px; cursor: pointer;';
 
+      const clearContainerBorder = () => {
+        const containerStyle = $container.getAttribute('style');
+        const newStyle = containerStyle
+          ?.replace('border: 1px dashed #6C6C6C;', '')
+          .replace('border: 1px dashed rgb(108, 108, 108)', '');
+        $container.setAttribute('style', newStyle as string);
+      };
+
       const dispatchNodeView = () => {
         if (typeof getPos === 'function') {
+          clearContainerBorder();
```

Comment:
We haven't been clearing the alignment button and border whenever `dispatchNodeView` is called.
Did you make this change intentionally for a specific reason?